src/core/mac/mac_frame.hpp

```diff
@@ -445,6 +453,23 @@ class Frame : public otRadioFrame
      */
     void SetSequence(uint8_t aSequence) { GetPsdu()[kSequenceIndex] = aSequence; }
 
+    /**
+     * This method indicates whether or not the Dst PanId is present.
+     *
+     * @param[in]  aFcf          The Frame Control field.
+     * @returns TRUE if the Dst PanId is present, FALSE otherwise.
```

Comment:
```suggestion
     * @param[in]  aFcf          The Frame Control field.
     
     * @returns TRUE if the Dst PanId is present, FALSE otherwise.
```